app/src/main/res/values/strings.xml

```diff
@@ -20,4 +20,6 @@
     <string name="GenericErrorMessage">Something went wrong. Please try again later.</string>
     <string name="GenericErrorButtonText">Retry</string>
     <string name="EditUserData">Edit User Data</string>
+    <string name="ProfileError">Something went wrong \nClick to refresh</string>
+    <string name="ProfileErrorButtonRetry">Retry</string>
```

Comment:
Yes, I agree with you. I added this string because I thought about the situation when we would like to change something on one screen and don't want to make an impact on the other, but now I will go with the generic one to omit to repeat the same value 😄. Also, I will rename it to `RetryButton` to make the name more meaningful and easier to adjust in other places.

Reply:
Done 😄